Anlab.Mvc/Controllers/FinancialServiceController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AnlabMvc.Services;
+using Microsoft.AspNetCore.Authorization;
+using Anlab.Jobs.MoneyMovement;
+
+namespace AnlabMvc.Controllers
+{
+    [Authorize]
+    public class FinancialServiceController : ApplicationController
+    {
+        private readonly IFinancialService _financialService;
+        
+        public FinancialServiceController(IFinancialService financialService)
+        {
+            _financialService = financialService;
+        }
+
+        [HttpGet("financial/{account}")]
+        public async Task<string> GetAccountInfo(string account)
+        {
+            var accountModel = new AccountModel(account);
+            var result = await _financialService.GetAccountInfo(accountModel.Chart, accountModel.Account);
```

Comment:
@laholstege throw in a little null checking after you create the account model, since there is no reason to call the financial service w/ a null or empty chart or account.  In that case returning null seems easiest, though empty string works too.  Also call it GetAccountName() for now since it's only the name that we are returning.

Reply:
@laholstege Only call if it is a uc davis account (Chart 3 or L). @srkirkland Or does this get info for other colleges?